README.md

```diff
@@ -366,6 +368,14 @@ Content-Type: application/json
 
 ### Global Variables
 Global variables provide a pre-defined set of variables that can be used in any part of the request(Url/Headers/Body) in the format `{{$variableName}}`. Currently, we provide a few dynamic variables which you can use in your requests. The variable names are _case-sensitive_.
+* `{{$aadToken [new|clear] [public|cn|de|us|ppe] [<domain|tenantId>]}}`: Add an Azure Active Directory token based on the following options (must be specified in order):
+

```

Comment:
I don't like adding the `clear` code action directly inside the document, which makes ambiguous with using a variable value or url. So how about add a command directly in vscode, or simply remove this option.

Reply:
Good idea. A command sounds like the right approach for this. I'll remove it for now and re-add it as a command later.